### PR TITLE
add test for HelloWorldServlet

### PR DESCRIPTION
**Pull request by reiniergs**

### CODE REVIEW

**Comment by reiniergs** [comment]
@jodarove I need review on this

File: src/test/java/HelloWorldTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        new HelloWorld().doGet(request, response);
+
+        verify(request, atLeast(1)).getParameter("thing");
+        writer.flush();
+        assertTrue(FileUtils.readFileToString(new File("somefile.txt"), "UTF-8")
+                .contains("Hello World"));
+    }
+
+    @After
```
Comment (jodarove) [inline src/test/java/HelloWorldTest.java]: you can remove this after using the stringWriter

File: src/test/java/HelloWorldTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        request = mock(HttpServletRequest.class);
+        response = mock(HttpServletResponse.class);
+    }
+
+    @Test
+    public void ShouldSayHelloToThingWhenThingIsPassed() throws Exception {
+
+        when(request.getParameter("thing")).thenReturn("Reinier");
+
+        PrintWriter writer = new PrintWriter("somefile.txt");
```
Comment (jodarove) [inline src/test/java/HelloWorldTest.java]: I don't like to have io on this test, you could do this:
```java
    @Test
    public void testCanUseStringWriterInsteadOfFile() {
        StringWriter writerOutput = new StringWriter();
        PrintWriter writer = new PrintWriter(writerOutput);

        writer.println("Hello World");

        assertTrue(writerOutput.toString().contains("Hello World"));
    }
```

File: pom.xml [hunk 3]
```diff
@@ -29,6 +29,18 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.8.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```
Comment (jodarove) [inline pom.xml]: is this still needed?

**Comment by jodarove** [comment]
+1 after removing the commons-io maven dependency

**Pull request merged**